cli/src/alluxio.org/cli/cmd/fs/fs.go

```diff
@@ -12,37 +12,40 @@
 package fs
 
 import (
```

Comment:
i think golang has strict lint rules... your code won't compile cc @Xenorith

Reply:
the rule isn't enforced but you could run this command to recursively format the go files: https://golang.cafe/blog/how-to-run-gofmt-recursively.html

i will likely iterate on these new commands and add docs similar to what was done in https://github.com/Alluxio/alluxio/pull/1814 so i am not too concerned about doing the cleanup now